code/src/components/InputCheckbox.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../Styles/InputCheckbox.css';
+
+const InputCheckbox = ({ question, array, selectedMonths, onMonthsChange }) => {
```

Comment:
I think it's goof you are naming you props here, makes the code easy to follow

Reply:
I think so too, much easier to read and change instead of writing  props. before. 😃